src/Robo/Plugin/Commands/ToolingCommands.php

```diff
@@ -106,4 +106,22 @@ protected function updateRoboConfig(string $key, string $value): void
         $roboConfig[$key] = $value;
         file_put_contents($roboConfigPath, Yaml::dump($roboConfig));
     }
+
+    /**
+     * Get Robo configuration value.
+     *
+     * @param string $key
+     *   The key of the configuration to load.
+     *
+     * @return mixed
+     *   A configuration value.
+     */
+    protected function getConfig(string $key)
+    {
+        $configValue = Robo::config()->get($key);
+        if (!isset($configValue)) {
```

Comment:
Wouldn't it always be set, but it could be empty?

Or does `get()` have a return type that makes this check work?

Reply:
Just kidding. PHPStan _hates_ empty. Switched back to isset() since it worked in my testing.